klangbecken.py

```diff
@@ -1041,9 +1043,16 @@ def playlog_cmd(data_dir, filename, off_air=False, dev_mode=False):
         json.dump(data, f, **json_opts)
         del data
 
+    # Update file metadata
+    FileType = SUPPORTED_FILE_TYPES[filename.split(".")[-1]]
```

Comment:
`SUPPORTED_FILE_TYPES` is defined with leading dots so you need to include that here as well (`FileType = SUPPORTED_FILE_TYPES["." + filename.split(".")[-1]]`).

Reply:
yep, it is a while ago... thanks